src/installer/hash.ts

```diff
@@ -72,6 +72,11 @@ async function getOnlineHash(clientFile: ClientFile): Promise<HashFileContents |
 
     // this is the hash kamel uses - the others use the sha256 txt file
     const md5Filename = `${clientFile.archiveFilename}.md5`;
+
+    // crda checksum file is crda_0.2.3_checksums.txt
+    const version = clientFile.version;
+    SHA_FILENAMES.push(`${clientFile.clientName}_${version.slice(1, version.length)}_checksums.txt`);
```

Comment:
rather than modifying the top-level array can you copy it.

```
const crdaVersionedShaFilename = `${clientFile.clientName}_${version.slice(1, version.length)}_checksums.txt`;
const crdaShaFilenames = [ ...SHA_FILENAMES, crdaVersionedShaFilename ];
```

Reply:
Okay, Thanks :+1: